packages/create-waku/src/index.ts

```diff
@@ -4,21 +4,45 @@ import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 import { parseArgs } from 'node:util';
 import { default as prompts } from 'prompts';
-import { red, green, bold } from 'kolorist';
+import { red, green, bold, cyan } from 'kolorist';
 import fse from 'fs-extra/esm';
 import checkForUpdate from 'update-check';
 import { createRequire } from 'node:module';
+import {
+  downloadAndExtractExample,
+  downloadAndExtractRepo,
+  existsInRepo,
+  getRepoInfo,
+  hasRepo,
+  type RepoInfo,
+} from './helpers/examples';
```

Comment:
The file name "examples" is a bit confusing. Maybe simply like this for now?
```suggestion
} from './helpers';
```

Reply:
how about this?

```
/helper
  examples.ts
  index.ts

// /helper/index.ts
export * from './examples'
```

or something else?

Reply:
that's worse. we don't use `index.ts` except for the root.

Reply:
```
$ ls **/index.ts
examples/02_demo/src/lib/index.ts	packages/website/src/hooks/index.ts
packages/create-waku/src/index.ts	packages/website/src/utils/index.ts
packages/website/src/atoms/index.ts
```
I don't particularly like them either.

We used to use cli.ts in create-waku, but there's a limitation with a bundler. cc @himself65

Reply:
I got it.
But I thought it's good to refactor for seperating the util functions into that `helpers` folder because the create-waku's `index.ts` is a bit large
this is why i make the helpers folder.
Do you want to make only `helpers.ts`?

Reply:
I'm fine with `helpers/*.ts`. It's just that "examples.ts" is confusing, just a bit. It can be read as "examples of helpers" instead of "helpers for examples."

Reply:
i see. i will rename `examples.ts` file to `example-option.ts`